app/src/main/kotlin/cz/covid19cz/erouska/service/CovidService.kt

```diff
@@ -113,12 +115,15 @@ class CovidService : Service() {
     private val wakeLockManager by inject<WakeLockManager>()
     private val powerManager by inject<PowerManager>()
     private val localBroadcastManager by inject<LocalBroadcastManager>()
+    private val alarmManager by inject<AlarmManager>()
     private val notificationManager = CovidNotificationManager(this)
     private val shortcutsManager = ShortcutsManager(this)
 
     private var bleAdvertisingDisposable: Disposable? = null
     private var bleScanningDisposable: Disposable? = null
 
+    private val autoRestartJob = AutoRestartJob()
```

Comment:
Btw je trosku divny takhle wrapovat tu logiku do toho broadcast receiveru a ten si pak instanciovat.

Reply:
Hele na jednu stranu jo, ale mně přišlo docela fajn to mít prostě celý pohromadě a vlastně říct "ty jsi auto-restart job, tak se nějak nastav/zruš" a to, že to je udělaný přes broadcast, tak už je vnitřní detail tý komponenty.

Reply:
Udelal bych na to asi spesl tridu.

Reply:
OK...